docs/troubleshooting.md

```diff
@@ -0,0 +1,351 @@
+# Uhura Troubleshooting Guide
+
+This document contains information that may help you troubleshoot issues you may encounter while installing, configuring and/or running Uhura.
+
+## Missing Environment Variables
+
+The **envied** gem ensures that the environment variables you specify in Envfile exist.
+
+If  you have not sourced your .env file in your development environment and attempt to start the Rails Console you may encounter the following:
+
+```
+$ bundle exec rails console
+DEPRECATION WARNING: Default values will be removed in the next minor-release of ENVied (i.e. > v0.9). For more info see https://gitlab.com/envied/envied/tree/0-9-releases#defaults. (called from block in load at /home/lex/Clients/Concur/Projects/uhura/Envfile:4)
+Traceback (most recent call last):
+	14: from bin/rails:4:in `<main>'
+	13: from bin/rails:4:in `require'
+	12: from /home/lex/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/railties-6.0.0.rc2/lib/rails/commands.rb:18:in `<top (required)>'
+	11: from /home/lex/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/railties-6.0.0.rc2/lib/rails/command.rb:46:in `invoke'
+	10: from /home/lex/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/railties-6.0.0.rc2/lib/rails/command/base.rb:65:in `perform'
+	 9: from /home/lex/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
+	 8: from /home/lex/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
+	 7: from /home/lex/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
+	 6: from /home/lex/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/railties-6.0.0.rc2/lib/rails/commands/console/console_command.rb:101:in `perform'
+	 5: from /home/lex/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/railties-6.0.0.rc2/lib/rails/command/actions.rb:14:in `require_application_and_environment!'
```

Comment:
We don't need to have your `/home/lex` in the README / troubleshooting. Can you just paste the command and the error ?